apps/test/unit/applab/AppLabTooltipOverlayTest.js

```diff
@@ -72,34 +64,35 @@ describe('AppLabTooltipOverlay', () => {
 
     beforeEach(() => {
       stubDraggedElementDropPoint.returns({ left: DROP_POINT_X, top: DROP_POINT_Y });
-      renderer.render(<AppLabTooltipOverlay {...TEST_PROPS} />);
-      result = renderer.getRenderOutput();
+      result = shallow(<AppLabTooltipOverlay {...TEST_PROPS} />);
     });
 
     it('renders with overridden position', () => {
-      expect(result.props.mouseX).to.equal(DROP_POINT_X);
-      expect(result.props.mouseY).to.equal(DROP_POINT_Y);
+      var props = result.find('TooltipOverlay').props();
+      expect(props.mouseX).to.equal(DROP_POINT_X);
+      expect(props.mouseY).to.equal(DROP_POINT_Y);
     });
 
     it('renders tooltip above cursor', () => {
-      expect(result.props.tooltipAboveCursor).to.be.true;
+      var props = result.find('TooltipOverlay').props();
+      expect(props.tooltipAboveCursor).to.be.true;
     });
 
     it('gives modified position to coordinates provider', () => {
-      expect(result.props.providers[0]).to.be.a('function');
-      expect(result.props.providers[0](result.props))
+      expect(result.props().providers[0]).to.be.a('function');
+      expect(result.props().providers[0](result.props()))
           .to.equal(`x: ${Math.floor(DROP_POINT_X)}, y: ${Math.floor(DROP_POINT_Y)}`);
     });
   });
 
   describe('when over an applab control', () => {
     const CONTROL_ID = 'fake-id';
     const SCREEN_ID = 'screen-id';
-    let fakeElement, fakeScreen;
+    let fakeElement, fakeScreen, fakeContainer;
```

Comment:
nit: you no longer need to expose fakeContainer here, since it didn't end up being necessary for the AppLabTooltipOverlay test